posthog/session_recordings/session_recording_api.py

```diff
@@ -559,21 +556,9 @@ def _gather_session_recording_sources(self, recording: SessionRecording) -> Resp
         blob_prefix = ""
 
         if recording.object_storage_path:
-            if recording.storage_version == "2023-08-01":
-                blob_prefix = recording.object_storage_path
-                blob_keys = object_storage.list_objects(cast(str, blob_prefix))
-            else:
-                # originally LTS files were in a single file
-                # TODO this branch can be deleted after 01-08-2024
-                sources.append(
-                    {
-                        "source": "blob",
-                        "start_timestamp": recording.start_time,
-                        "end_timestamp": recording.end_time,
-                        "blob_key": recording.object_storage_path,
-                    }
-                )
-                might_have_realtime = False
+            blob_prefix = recording.object_storage_path
+            blob_keys = object_storage.list_objects(cast(str, blob_prefix))
+            might_have_realtime = False
```

Comment:
there was a bug here, that we didn't set might_have_realtime as false for these recordings

good job it didn't take a year to spot it 🙈